src/scripts/quest/subquest/limsa/SubSea012.cpp

```diff
@@ -0,0 +1,141 @@
+#include <Script/NativeScriptApi.h>
+#include <Actor/Player.h>
+#include "Manager/EventMgr.h"
+#include <ScriptObject.h>
+#include "Framework.h"
+
+using namespace Sapphire;
+
+// Quest Script: SubSea012_00122
+// Quest Name: Glory Days
+// Quest ID: 65658
+// Start NPC: 1003601
+// End NPC: 1000972
+
+class SubSea012 :
+  public Sapphire::ScriptAPI::EventScript
+{
+private:
+
+  // Basic quest information
+  // Quest vars / flags used
+  // GetQuestUI8AL
+  // GetQuestUI8BH
+
+  enum Sequence :
+    uint8_t
+  {
+    Seq0 = 0,
+    Seq1 = 1,
+    Seq2 = 2,
+    SeqFinish = 255,
+  };
+
+  // Quest rewards 
+  static constexpr auto RewardExpFactor = 100;
+  // Entities found in the script data of the quest
+  static constexpr auto Actor0 = 1003601;
+  static constexpr auto Actor1 = 1001024;
+  static constexpr auto Actor2 = 1000972;
+  static constexpr auto Enemy0 = 563;
+  static constexpr auto Item0 = 2000454;
+
+public:
+  SubSea012() :
+    Sapphire::ScriptAPI::EventScript(65658)
+  {
+  };
+
+  ~SubSea012()
+  {
+  };
+
+  void onTalk(uint32_t eventId, Entity::Player& player, uint64_t actorId) override
+  {
+    auto pEventMgr = m_framework->get< World::Manager::EventMgr >();
+    auto actor = pEventMgr->mapEventActorToRealActor( static_cast<uint32_t>(actorId) );
+
+    if (actor == Actor0)
```

Comment:
```suggestion
    if( actor == Actor0 )
```
should be like this, same for other conditionals